OracleWebLogic/README.md

```diff
@@ -2,6 +2,8 @@ WebLogic on Docker
 ===============
 This repository contains sample Docker configurations to facilitate installation, configuration, and environment setup for DevOps users. This project includes quick start [Dockerfiles](dockerfiles/) and [samples](samples/) for WebLogic Server 12.1.3, 12.2.1, 12.2.1.1, 12.2.1.2, and 12.2.1.3 based on Oracle Linux and Oracle JDK 8 (Server).
 
+**IMPORTANT**: The samples in this repository are for advanced use cases only. We provide Dockerfiles as samples to build WebLogic images but this is _NOT_ a recommended practice. We suggest alternative methods including obtaining base WebLogic Server images from the [Oracle Container Registry](https://oracle.github.io/weblogic-kubernetes-operator/userguide/base-images/ocr-images/), using the open source [WebLogic Image Tool](https://oracle.github.io/weblogic-kubernetes-operator/userguide/base-images/custom-images/) to create custom images, and using the open source [WebLogic Kubernetes Operator](https://oracle.github.io/weblogic-kubernetes-operator/) to deploy and manage WebLogic domains. For more information, see [Obtaining, Creating, and Updating Oracle Fusion Middleware Images with Patches](https://docs.oracle.com/en/middleware/fusion-middleware/12.2.1.4/opatc/obtaining-creating-and-updating-oracle-fusion-middleware-images-patches.html#GUID-4FB15429-C985-472F-BDC6-669CA1B678E8).
```

Comment:
Lets add we recommend obtaining patched WebLogic Server images, these images are patched with the latest security patches.